mypy.ini

```diff
@@ -15,8 +15,8 @@ disallow_untyped_defs = true
 disallow_incomplete_defs = true
 show_error_codes = false
 disable_error_code = empty-body
+python_version = 3.11
```

Comment:
`python_version` also determines the value of `sys.version_info` used when interpreting typeshed stubs (for conditionals such as https://github.com/python/typeshed/blob/main/stdlib/typing.pyi#L93-L113)

So if we override this setting, we lose test coverage with different variations of `typeshed` definitions, and would no longer catch subtle compatibility issues in CI.

I very much prefer the modern formatting of types in error messages, but still I would err on the side of caution and prioritize test coverage here.

I wish mypy had a way to override `force_uppercase_builtins` in the opposite direction, e.g. `force_lowercase_builtins`. Maybe they would be willing to entertain such a PR if someone is interested doing the work. And same for `force_union_syntax`.

Reply:
Yes, I agree - we cannot touch `python_version` :(